src/Http/Service/EventSubscriber/ViewResponseSubscriber.php

```diff
@@ -41,21 +44,28 @@ public function buildView(ViewEvent $event): void
     {
         $data = $event->getControllerResult();
 
-        if ($data instanceof ResourceCollectionResponse) {
+        if ($data instanceof CollectionResourceResponse) {
+            $responseLinkBuilder = new ResponseLinksBuilder(
```

Comment:
Why not to pull this from container ?

Reply:
Because laziness? :D Good idea, thanks for the tip